collector/exporter/otelarrowexporter/internal/arrow/exporter.go

```diff
@@ -251,6 +259,7 @@ func (e *Exporter) SendAndWait(ctx context.Context, data any) (bool, error) {
 
 // Shutdown returns when all Arrow-associated goroutines have returned.
 func (e *Exporter) Shutdown(_ context.Context) error {
+	close(e.shutdown)
```

Comment:
Basic question: It seems this sets the error to `context.Canceled` but wouldn't the next line `e.cancel()` also do this by canceling the context passed to [exporter.start()](https://github.com/open-telemetry/otel-arrow/blob/d6d28861e5fff1e80a2a61a17e8a240f13c2dcf2/collector/exporter/otelarrowexporter/internal/arrow/exporter.go#L136). What does `close(e.shutdown)` provide that `e.cancel()` is neglecting?

Reply:
I had added this to help fix the flakey test, but I later deleted the flakey test and wrote a new one. I think, at this point there is no real benefit to this change, so I've removed the shutdown field and its use. The tests pass w/o it -- removed.

However, there's an answer to your question -- the context passed to `SendAndWait()` is out of our control. If it happens to not be canceled and a shutdown is happening, it will block forever once the streams all exit. However, that doesn't stop this component from shutting down, it just presents a risk of flakey tests hanging.